android/hello_sdl_android/src/main/java/com/sdl/hellosdlandroid/SdlService.java

```diff
@@ -199,19 +199,44 @@ public void onError(String info, Exception e) {
 
 				@Override
 				public LifecycleConfigurationUpdate managerShouldUpdateLifecycle(Language language){
```

Comment:
Just to keep up with good formatting to keep all code consistent could you please add a space before the curly brace?

```
managerShouldUpdateLifecycle(Language language) {
```